content/blog/2019-08-01-new-react-devtools.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Introducing the new React DevTools"
+author: [bvaughn]
+---
+We are excited to announce a new release of the React developer tools, available today in supported browsers!
+
+## What's changed?
+
+A lot has changed!
+[Check out the changelog](https://github.com/bvaughn/react-devtools-experimental/blob/master/CHANGELOG.md) for a complete list.
+At a high level, this new version should offer significant performance gains and an improved navigation experience.
+
+## Which versions of React are supported?
+
+**`react-dom`**
+
+* `0`-`14.x`: Not supported
+* `15.x`: Partial (component filters not supported)
```

Comment:
This looks like a downgrade. But component filters are a new feature. Maybe there's another way to phrase it to make it look additive rather than diminishing.

Reply:
Good point.